### PR TITLE
feat : Authentication functionality

### PR DESCRIPTION
**Pull request by anuj-kumary**
**Authentication Functionality**
  - Implemented Signup and Signin 

ScreenShot:-
Screenshot from 2022-03-24 18-41-52
Screenshot from 2022-03-24 18-42-11

### CODE REVIEW

File: src/pages/Auth/Signup.jsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  const [signupForm, setSignupForm] = useState({
+    email: '',
+    password: '',
+    name: '',
+  });
+
+  const signupHandler = (e) => {
+    e.preventDefault();
+    const { email, password, name } = signupForm;
+    if (email && password && name !== '') {
```
Comment (Shah-Arsalan) [inline src/pages/Auth/Signup.jsx:17]: You can do this validation with required attribute only .

**Review by Shah-Arsalan — APPROVED** [review]
Code looks neat . You can go ahead and merge this branch

File: .history/src/components/Navbar/Navbar_20220324130532.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+export const Navbar = () => {
+  return (
+    <>
+      <nav className='navigation'>
+        <div className='navigation__logo'>
+          <h3 className='navigation__heading'>EagleNote</h3>
+        </div>
+        <ul className='navbar__social'>
+          <Link to='/login' className='navbar__social-link'>
```
Comment (Rohanmond) [inline .history/src/components/Navbar/Navbar_20220324130532.jsx]: you could use useNavigate here

**Review by Rohanmond — APPROVED** [review]

File: src/pages/Auth/Signup.jsx (hunk 1 above)
Comment (anuj-kumary) [reply src/pages/Auth/Signup.jsx:17]: Done

File: .history/src/components/Navbar/Navbar_20220324130532.jsx (hunk 2 above)
Comment (anuj-kumary) [reply .history/src/components/Navbar/Navbar_20220324130532.jsx]: I'll make changes

**Review by vinc3nati — APPROVED** [review]

**Pull request merged**